identity/handler_test.go

```diff
@@ -457,6 +457,46 @@ func TestHandler(t *testing.T) {
 				})
 			}
 		})
+		t.Run("case=should update an identity with credentials", func(t *testing.T) {
+			i := &identity.Identity{Traits: identity.Traits(fmt.Sprintf(`{"subject":"%s"}`, x.NewUUID().String()))}
+			require.NoError(t, reg.PrivilegedIdentityPool().CreateIdentity(context.Background(), i))
+
+			for name, ts := range map[string]*httptest.Server{"public": publicTS, "admin": adminTS} {
+				t.Run("endpoint="+name, func(t *testing.T) {
+					credentials := identity.AdminIdentityImportCredentials{
+						Password: &identity.AdminIdentityImportCredentialsPassword{
+							Config: identity.AdminIdentityImportCredentialsPasswordConfig{
+								Password: "pswd",
+							},
+						},
+					}
+					ur := identity.AdminUpdateIdentityBody{
+						Traits:         []byte(`{"bar":"baz","foo":"baz"}`),
+						SchemaID:       i.SchemaID,
+						State:          identity.StateInactive,
+						MetadataPublic: []byte(`{"public":"metadata"}`),
+						MetadataAdmin:  []byte(`{"admin":"metadata"}`),
+						Credentials:    &credentials,
+					}
+
+					res := send(t, ts, "PUT", "/identities/"+i.ID.String(), http.StatusOK, &ur)
+					assert.EqualValues(t, "baz", res.Get("traits.bar").String(), "%s", res.Raw)
```

Comment:
The test doesn't check whether the credentials have been actually updated - could you please add such an assertion? It probably makes sense to use `reg.PrivilegedIdentityPool().GetIdentity()` and then check the contents of `.Credentials`

Reply:
Done